test/test_redis_utils.py

```diff
@@ -66,3 +68,23 @@ def build_item_metadata():
         assert res['dirty'] == '0'
         assert res['item_type'] == 'Sample'
         assert json.loads(res['properties']) == obj
+
+    def test_redis_set(self, redisdb):
+        """ Tests some simple sets with expiration """
+        rd = RedisBase(redisdb)
+        my_key_meta = 'snovault:items:uuid:meta'
+        rd.set(my_key_meta, 'hello')
+        assert rd.get(my_key_meta) == 'hello'
+        rd.delete(my_key_meta)
+        # test exp with datetime
+        exp = datetime.timedelta(seconds=2)
+        rd.set(my_key_meta, 'hello', exp=exp)
+        assert rd.get(my_key_meta) == 'hello'
+        time.sleep(3)
```

Comment:
Is it possible to be using ControlledTime in this test to avoid having to have tests take a long time?

Reply:
I spoke to you on Slack, and you said not. Oh well.